tests/testthat/_snaps/ard_dichotomous.md

```diff
@@ -5,23 +5,7 @@
     Output
       [1] "card"       "tbl_df"     "tbl"        "data.frame"
 
----
-
-    Code
-      as.data.frame(dplyr::select(ard_dich, -c(fmt_fun, warning, error)))
-    Output
-        variable variable_level     context stat_name stat_label    stat
-      1      cyl              4 dichotomous         n          n      11
-      2      cyl              4 dichotomous         N          N      32
-      3      cyl              4 dichotomous         p          % 0.34375
-      4       am           TRUE dichotomous         n          n      13
-      5       am           TRUE dichotomous         N          N      32
-      6       am           TRUE dichotomous         p          % 0.40625
-      7     gear              3 dichotomous         n          n       5
-      8     gear              3 dichotomous         N          N      32
-      9     gear              3 dichotomous         p          % 0.15625
-
----
+# ard_dichotomous() errors are correct
```

Comment:
This test was deleted because of the change in R-devel, right?

Reply:
Yes - it will come back once the difference is resolved and the `skip_if_not()` is removed.